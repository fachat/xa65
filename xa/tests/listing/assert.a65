        *=$0400

        .assert *<$17e6, "routine too long"

        lda #<$17e6 >> 2

        lda *-2

