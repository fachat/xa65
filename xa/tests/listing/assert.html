<html><head><title>(null)</title></head><body><pre>

assert.a65

    1 A:1000                                     *= $0400

    3 A:0400                                     .assert *&lt;$17e6,"routine too long"

    5 A:0400  a9 f9                              lda #&lt;$17e6&gt;&gt;2

    7 A:0402  ad 00 04                           lda *-2
</pre></body></html>
