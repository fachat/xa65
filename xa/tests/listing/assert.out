
assert.a65

    1 A:1000                                     *= $0400

    3 A:0400                                     .assert *<$17e6,"routine too long"

    5 A:0400  a9 f9                              lda #<$17e6>>2

    7 A:0402  ad 00 04                           lda *-2
